.github/workflows/build_develop.yml

```diff
@@ -34,28 +38,81 @@ jobs:
           SENTRY_URL: ${{ secrets.SENTRY_URL }}
           SENTRY_ORG: element
           SENTRY_PROJECT: riot-web
+          # We have to update the CSP until Cloudflare Pages supports proxying redirects
```

Comment:
please could you expand on this? Why do we need to add R2_PUBLIC_URL to the CSP? Once CFP supports proxying redirects, how does that help?